doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md

```diff
@@ -0,0 +1,1332 @@
+  Laser Temperature High Warning : False
+  Laser Temperature Low Alarm : False
+  Laser Temperature Low Warning : False
+  ```
+- Example (Display pm status of transceiver connected to Ethernet0):
+  ```
+  admin@sonic:~$ show interfaces transceiver pm Ethernet0
+  Ethernet0:
+  prefec_ber_avg : 1.00E-3
+  prefec_ber_min : 1.00E-3
+  prefec_ber_max : 1.00E-3
+  uncorr_frames_avg : 0.00E0
+  uncorr_frames_min : 0.00E0
+  uncorr_frames_max : 0.00E0
+  cd_avg : 0 ps/ns
+  cd_min : 0 ps/ns
+  cd_max : 0 ps/ns
+  dgd_avg : 1.0 ps
+  dgd_min : 1.0 ps
+  dgd_max : 1.0 ps
+  sopmd_avg : 1 ps^2
+  sopmd_min : 1 ps^2
+  sopmd_max : 1 ps^2
+  pdl_avg  : 1.0 dB
+  pdl_min  : 1.0 dB
+  pdl_max  : 1.0 dB
+  osnr_avg : 28.0 dB
+  osnr_min : 28.0 dB
+  osnr_max : 28.0 dB
+  esnr_avg : 15.0 dB
+  esnr_min : 15.0 dB
+  esnr_max : 15.0 dB
+  cfo_avg  : 500 MHz
+  cfo_min  : 500 MHz
+  cfo_max  : 500 MHz
+  soproc_avg : 1 krad/s
+  soproc_min : 1 krad/s
+  soproc_max : 1 krad/s
+  tx_power_avg : -10.00 dBm
+  tx_power_min : -10.00 dBm
+  tx_power_max : -10.00 dBm
+  rx_tot_power_avg : -8.00 dBm
+  rx_tot_power_min : -8.00 dBm
+  rx_tot_power_max : -8.00 dBm
+  rx_sig_power_avg : -8.00 dBm
+  rx_sig_power_min : -8.00 dBm
+  rx_sig_power_max : -8.00 dBm
+  ```
+  
+#### 2.3 Config_DB Schema ####
+##### 2.3.1 Transceiver Config Table #####
+Stores information for physical switch ports managed by the switch chip. Ports to the CPU (ie: management port) and logical ports (loopback) are not declared in the PORT_TABLE. See INTF_TABLE.
+
+    ;Defines layer 2 ports
+    ;In SONiC, Data is loaded from configuration file by portsyncd
+    key                 = PORT_TABLE:ifname     ; ifname must be unique across PORT,INTF,VLAN,LAG TABLES
+    admin_status        = "down" / "up"         ; admin status
+    oper_status         = "down" / "up"         ; oper status
+    lanes               = list of lanes         ; (need format spec???)
+    mac                 = 12HEXDIG              ;
+    alias               = 1*64VCHAR             ; alias name of the port used by LLDP and SNMP, must be unique
+    description         = 1*64VCHAR             ; port description
+    speed               = 1*6DIGIT              ; port line speed in Mbps
+    mtu                 = 1*4DIGIT              ; port MTU
+    fec                 = 1*64VCHAR             ; port fec mode
+    lpmode              = "enable" / "disable"  ; port low power mode
+    configured_freq     = 1*9DIGIT              ; configured frequency
+    configured_TX_power = FLOAT                 ; configured TX output power 
+    loopback            = "media output" / "media input" / "host output" / "host input" / "none" ; loopback mode
+    
+    
+#### 2.4 Configure interfaces transceiver CLI
+configure privisioning settings of the transceivers
+
+- Usage:
+    ```
+    configure interfaces transceiver [<interface_name>] (lpmode  | configured_frequency | configured_tx_power | loopback) 
```

Comment:
configured_frequency -> frequency

configured_tx_power -> tx_power

Reply:
Done with the change.